src/watchFixPlugin/__tests__/index.test.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable global-require */
+const chalk = require('chalk');
+
+jest.doMock('chalk', () => new chalk.constructor({ enabled: false }));
+
+jest.useFakeTimers();
+
+let WatchFixPlugin;
+let configOverrides;
+
+const runAndAwait = async plugin => {
+  const runPromise = plugin.run();
+  await jest.runTimersToTime(700);
```

Comment:
we have `runAllPendingTimers` and `runAllTimers` instead of hardcoding the values

Reply:
They also don't return promises, so you can drop the `await`